### ISSUE TITLE
Cannot publish 'evt.comp.task.list', session is not yet established

### ISSUE THREAD

**Issue by Slavenin** [golemfactory/clay#1877]
Hi!
In my log file many errors like
```
2018-01-10 16:28:43 WARNING  golem.rpc                           b"RPC: Cannot publish 'evt.comp.task.list', session is not yet established"
2018-01-10 16:28:43 WARNING  golem.rpc                           b"RPC: Cannot publish 'evt.net.connection', session is not yet established"
2018-01-10 16:28:43 WARNING  golem.rpc                           b"RPC: Cannot publish 'evt.pay.balance', session is not yet established"
2018-01-10 16:28:44 INFO     golem.pay                           b'Peer count: 25 '
2018-01-10 16:28:44 INFO     golem.network.p2p.p2pservice        b'Adding peer '30f00040ffcf8579f48f436eaea5f2f2e45c70bb5cb7cfe83f493aeeaa09851aa003382bbb278e34e3b0469bcbf092bec34c0dd44cfabf094c0ff6ecb6a4fad3', key id difficulty: 0'
2018-01-10 16:28:44 INFO     golem.pay                           b'ETH: 0.01901288 '
2018-01-10 16:28:44 INFO     golem.pay                           b'GNT: 1000.0 '
```

What does it mean?

**Comment by badb** [comment]
Hi @Slavenin!

It means that there are some problems with crossbar router which connects golem core with user interface. Do you know when do these errors occur? Were you able to start application normally? Do they start to appear at the beginning or after some time or just after closing golem app?

**Comment by Slavenin** [comment]
Application start  normally. After start open one window 
![image](https://user-images.githubusercontent.com/4180077/34811575-8f8b43c8-f6b2-11e7-8978-b9883e6190a0.png) . Started at 2018-01-10 16:01:52 first error in 2018-01-10 16:19:53. I attached full log in this message. After close all golem processes stop. But VB is work 
![image](https://user-images.githubusercontent.com/4180077/34811683-15793896-f6b3-11e7-9ba2-a78c232f3322.png)


[golem.log](https://github.com/golemfactory/golem/files/1621490/golem.log)

**Comment by golemUser1** [comment]
I have reistalled golem today (i had to correct a mess involving a vmware update, and a no longer working docker configuration) and run in the same issue afterwards: 

I published a test task from http://golem.timjones.id.au/.

[golem.log](https://github.com/golemfactory/golem/files/1624247/golem.log)

At the beginning everything looks normal. (i have started the logfile after resolving all issues with docker.) 

Accidently i have to pay 56 golem for a Test job of seconds.... (but thats another issue... )

After the job was complete the issue happened, directly after submitting the payment. so it happend in the middle of runnning.
in comparison to Slavenin's Log the cause event might be:

> 2018-01-11 22:25:57 WARNING  twisted                             b'session closed with reason wamp.close.transport_lost [()] '

I noticed it because the payment in the history tab was not actualised; so i looked into the log file.

The golem app seems working again after a restart.

the following critial entry was a direct consequence of this issue, it might also be worth a look:
> 2018-01-11 22:27:07 CRITICAL twisted                             b'Unhandled Error\nTraceback (most recent call last):\n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\twisted\\python\\log.py", line 103, in callWithLogger\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\twisted\\python\\log.py", line 86, in callWithContext\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\twisted\\python\\context.py", line 122, in callWithContext\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\twisted\\python\\context.py", line 85, in callWithContext\n    \n--- <exception caught here> ---\n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\twisted\\internet\\iocpreactor\\reactor.py", line 119, in _callEventCallback\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\twisted\\internet\\iocpreactor\\tcp.py", line 299, in cbConnect\n    \nbuiltins.AttributeError: \'Client\' object has no attribute \'socket\'\n '

i just found issue:  #1591 and #1580 ; this seems similar.

**Comment by golemUser1** [comment]
It happenend again today: see log excerpt:

> 
> 2018-01-13 00:31:51 WARNING  twisted                             b'dropping connection to peer process:19148 with abort=True: WebSocket ping timeout (peer did not respond with pong in time) '
> 2018-01-13 00:31:51 WARNING  twisted                             b'session closed with reason wamp.close.transport_lost [()] '
> 2018-01-13 00:31:51 INFO     twisted                             b'Scheduling retry 1 to connect <twisted.internet.endpoints.TCP4ClientEndpoint object at 0x0000026FB8EFC978> in 2.023618257814788 seconds. '
> 2018-01-13 00:31:51 INFO     golem.network.p2p.peersession       b'Starting peer session '173.212.219.99':40102'
> 2018-01-13 00:31:51 WARNING  golem.rpc                           b"RPC: Cannot publish 'evt.comp.task.list', session is not yet established"
> 2018-01-13 00:31:52 INFO     twisted                             b'Native worker connection closed cleanly. '
> 2018-01-13 00:31:52 INFO     twisted                             b'Node worker worker-001 ended successfully '
> 2018-01-13 00:31:52 INFO     twisted                             b"Checking for node shutdown: worker_exit_success=True, shutdown_requested=False, node_shutdown_triggers=['shutdown_on_shutdown_requested'] "
> 2018-01-13 00:31:52 INFO     twisted                             b'Node will continue to run! '
> 2018-01-13 00:31:52 WARNING  golem.rpc                           b"RPC: Cannot publish 'evt.pay.balance', session is not yet established"
> 2018-01-13 00:31:54 WARNING  golem.task.taskserver               b'Impossible to start session with Node Wanted, (key: 7fbde5cdc029ac8019066175958d34c32bb76ea54b28a8e13c5f0884abac4b16d1f15920f01c18264a2dab7f837d34a741fe672c169e5ed688f918f1b29e3f4d)'
> 2018-01-13 00:31:54 WARNING  golem.task.taskserver               b'Impossible to start session with Node MPJB, (key: f0ab7a8bbf3ccde47ae2c572e7064a872a273a94c1c04ad692b3b81ea508b0154078f60c74b77ed141903e062461bb6b149d682dc1096b1b76a4f815607fe9bb)'
> 2018-01-13 00:31:54 WARNING  golem.task.taskserver               b'Impossible to start session with Node MPJB, (key: f0ab7a8bbf3ccde47ae2c572e7064a872a273a94c1c04ad692b3b81ea508b0154078f60c74b77ed141903e062461bb6b149d682dc1096b1b76a4f815607fe9bb)'
> 2018-01-13 00:31:55 WARNING  golem.rpc                           b"RPC: Cannot publish 'evt.comp.task.list', session is not yet established"

**Comment by badb** [comment]
This seems to be dying RPC issue that should be solved in 0.11.0. Feel free to comment if you observe this behaviour in current releases!
